### PR TITLE
Added empty path checking func and adjusted some of the root.go code

### PR DESCRIPTION
**Pull request by arian-amador**
Added some export comments to the exported vars, removed some extra
blank lines, and made small changes to the initEtcHosts func.

Also added go modules to remove the GOPATH dependency.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module github.com/arian-amador/txeh
```
Comment (cjimti) [inline go.mod]: I believe this should be github.com/txn2/txeh

**Comment by cjimti** [comment]
@arian-amador nice updates. Once I've tested the update to go modules I'll remove glide. It does look like the go module needs to be github.com/txn2/txeh but you have github.com/arian-amador/txeh. 

Thanks!

**Comment by arian-amador** [comment]
@cjimti you're absolutely right. I fixed it on my `master` and will have to watch out for that in the future :+1:

**Comment by cjimti** [comment]
@arian-amador thank you for your contribution!

**Pull request merged**